.travis.yml

```diff
@@ -10,6 +10,7 @@ env:
   global:
     - TERRAFORM_VERSION=0.9.4
     - PACKER_VERSION=1.1.2
+    - PIP=9.0.3
```

Comment:
Related to issues introduced by the last pip version. Rolled back to a stable one.